faidx.c

```diff
@@ -212,8 +213,10 @@ static faidx_t *fai_read(hFILE *fp, const char *fname)
     if (!buf) goto fail;
 
     while ((l = hgetln(buf, 0x10000, fp)) > 0) {
-        for (p = buf; *p && isgraph_c(*p); ++p);
-        *p = 0; ++p;
+        for (p = buf; *p && !isspace(*p); ++p);
```

Comment:
I don't think it's necessary these days.  The annoying warnings that it's trying to suppress seem to have gone away these days - presumably thanks to a more sensible implementation of `isspace()` etc. in libc.